src/tools/kdb/mountpoint-mount.c

```diff
@@ -45,51 +43,191 @@ void addMountSpec (KeySet * spec)
+	/* TODO: Decide where to put code for evaluation of option (currently also in cmerge.c) */
+	const char * optStrategy = NULL;
+	/* TODO: Implement option */
+	const char * optResolver = NULL;
+	int mergeStrategy = MERGE_STRATEGY_ABORT;
+
+	/* 3. Command arguments ("[path mountpoint] [plugin [config] [..]]" */
+	/* current simplified parameters: "kdb mount [options] [path mountpoint] [plugin]" */
+	const char * argPath = NULL;
+	const char * argMountpoint = NULL;
+
+	const Key * tmp;
+	if ((tmp = GET_OPTION_KEY (options, "debug")))
+		elektraKeyToBoolean (tmp, &optDebug);
+	if ((tmp = GET_OPTION_KEY (options, "verbose")))
+		elektraKeyToBoolean (tmp, &optVerbose);
+	if ((tmp = GET_OPTION_KEY (options, "version")))
+		elektraKeyToBoolean (tmp, &optVersion);
+	if ((tmp = GET_OPTION_KEY (options, "nonewline")))
+		elektraKeyToBoolean (tmp, &optSuppressNewline);
+
+	/* command-specific options */
+	if ((tmp = GET_OPTION_KEY (options, "force")))
+		elektraKeyToBoolean (tmp, &optForce);
+	if ((tmp = GET_OPTION_KEY (options, "quiet")))
+		elektraKeyToBoolean (tmp, &optQuiet);
+	if ((tmp = GET_OPTION_KEY (options, "interactive")))
+		elektraKeyToBoolean (tmp, &optInteractive);
+	if ((tmp = GET_OPTION_KEY (options, "with-recommends")))
+		elektraKeyToBoolean (tmp, &optWithRecommends);
+
+	/* TODO: Remove code duplication (code for processing strategy-parameter is also present in cmerge.c */
+	if ((tmp = GET_OPTION_KEY (options, "strategy")))
+	{
+		elektraKeyToString (tmp, &optStrategy);
+		if (elektraStrCmp (optStrategy, "our") == 0)
+		{
+			mergeStrategy = MERGE_STRATEGY_OUR;
+		}
+		else if (elektraStrCmp (optStrategy, "their") == 0)
+		{
+			mergeStrategy = MERGE_STRATEGY_THEIR;
+		}
+		else if (elektraStrCmp (optStrategy, "abort") != 0)
+		{
+			ELEKTRA_SET_VALIDATION_SEMANTIC_ERRORF (errorKey, "'%s' is not a valid strategy.", optStrategy);
+			return -1;
+		}
+	}
+
+	/* Command arguments */
+	if ((argPath = GET_OPTION(options, "path")) == NULL)
 	{
-		elektraKeyToBoolean (GET_OPTION_KEY (options, "verbose"), &verbose);
+		/* no path specified */
+		/* TODO: Decide if list mountpoints instead of returning error code */
+		return -1;
 	}
 
-	const char * path = GET_OPTION (options, "path");
+	if ((argMountpoint = getKeyNameFromOptions (options, GET_OPTION(options, "mountpoint"), errorKey, optVerbose)) == NULL)
+	{
+		//TODO: Error handling (mountpoint is not a valid keyname)
+		return -1;
+	}
 
-	const char * mountpoint = getKeyNameFromOptions (options, GET_OPTION (options, "mountpoint"), errorKey, verbose);
-	if (mountpoint == NULL) return 1;
+	Key * keyMp = keyNew (argMountpoint, KEY_END);
+	elektraNamespace keyMpNs = keyGetNamespace (keyMp);
+	if (!optQuiet && *argPath == '/' && keyMpNs != KEY_NS_SYSTEM && keyMpNs != KEY_NS_SPEC && keyMpNs != KEY_NS_CASCADING)
+	{
+		printf("Note that absolute paths are still relative to their namespace (see `kdb plugin-info resolver`).\n");
+		printf("Only system+spec mountpoints are actually absolute.\n");
```

Comment:
That still depends on the backend plugin and for `backend` on resolver plugin. It's just that the default `resolver` does it this way